src/python/pants/testutil/option_util.py

```diff
@@ -16,7 +16,7 @@ def create_options_bootstrapper(
     args: Iterable[str] | None = None, *, env: Mapping[str, str] | None = None
 ) -> OptionsBootstrapper:
     return OptionsBootstrapper.create(
-        args=("--pants-config-files=[]", *(args or [])),
+        args=("pants", "--pants-config-files=[]", *(args or [])),
```

Comment:
This seems like not a small issue - OptionsBootstrapper.create() expects the first arg to be the binary name. This means that these tests were instead ignoring the `--pants-config-files=[]` value. This didn't break the tests in practice, but led to a discrepancy in the option values vs the Rust parser.